### ISSUE TITLE
CUSA14974 - Wargroove

### ISSUE THREAD

**Issue by rambass658** [shadps4-emu/shadps4-game-compatibility#594]
### Game Name

Wargroove

### Game code

CUSA14974

### Game version

01.00

### Used emulator's version (only released versions are acceptable)

0.2.0

### Current status

Nothing

### Error

[Lib.SysModule] <Error> sysmodule.cpp:sceSysmoduleLoadModule:47: (DUMMY) called module = ORBIS_SYSMODULE_NP_AUTH
[Lib.SysModule] <Error> sysmodule.cpp:sceSysmoduleLoadModule:47: (DUMMY) called module = ORBIS_SYSMODULE_NP_TROPHY
[Lib.SysModule] <Error> sysmodule.cpp:sceSysmoduleLoadModule:47: (DUMMY) called module = ORBIS_SYSMODULE_APP_CONTENT

[Core.Linker] <Error> linker.cpp:InitTlsForThread:346: TLS user malloc called, using std::malloc
[Core.Linker] <Error> linker.cpp:InitTlsForThread:346: TLS user malloc called, using std::malloc
[Core.Linker] <Error> linker.cpp:InitTlsForThread:346: TLS user malloc called, using std::malloc
[Core.Linker] <Error> linker.cpp:InitTlsForThread:346: TLS user malloc called, using std::malloc

### Description

Crash.

### Screenshots

![image](https://github.com/user-attachments/assets/6e37db87-7419-47bb-ae30-3f60bf330b2c)

### Log File

[shad_log.txt](https://github.com/user-attachments/files/17017337/shad_log.txt)

**Comment by StevenMiller123** [comment]
Your log doesn't contain the error, and the image covers the stub. Could you make sure your logType is set to sync?

**Comment by rambass658** [comment]
Now I'll try to run it again. I'll be back soon.

**Comment by rambass658** [comment]
> Your log doesn't contain the error, and the image covers the stub. Could you make sure your logType is set to sync?

Yeah, i set sync. [New log.](https://github.com/user-attachments/files/17017517/shad_log.txt)

**Comment by StevenMiller123** [comment]
So it's crashing beacause libkernel's _is_signal_return function is unimplemented. Thanks for the report.

**Comment by mikusp** [comment]
The same error happens on 0.3.0

**Comment by Crispy81** [comment]
0.4 crashes and does nothing.

![image](https://github.com/user-attachments/assets/4ee11e8e-26a5-49d8-a682-3fec24fe269e)

[shad_log.txt](https://github.com/user-attachments/files/17898956/shad_log.txt)
